pkg/model/database_secret.go

```diff
@@ -17,8 +17,8 @@ func DatabaseSecret(cr *v1alpha1.Keycloak) *v1.Secret {
 			},
 		},
 		StringData: map[string]string{
-			DatabaseSecretUsernameProperty: cr.ObjectMeta.Name + "-" + RandStringRunes(4),
-			DatabaseSecretPasswordProperty: cr.ObjectMeta.Name + "-" + RandStringRunes(4),
+			DatabaseSecretUsernameProperty: cr.ObjectMeta.Name + "-" + GenerateRandomString(4),
```

Comment:
I would just hardcode the username to something like "keycloak"

Reply:
Fixed.